extension/persistence/eclipselink/build.gradle.kts

```diff
@@ -33,6 +33,7 @@ dependencies {
   implementation(libs.eclipselink)
   implementation(platform(libs.dropwizard.bom))
   implementation("io.dropwizard:dropwizard-jackson")
+  implementation("org.postgresql:postgresql:42.6.0")
```

Comment:
postgres (and all DBs) has been intentionally left out of the eclipselink build.

Also, this seems unrelated to the title of the PR